ansible/smoke.yml

```diff
@@ -150,3 +150,28 @@
       become: false
       run_once: true
       when: smoke_enable_cdn_checks | default(True)
+
+- name: catalog-next.data.gov /dataset redirection
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: assert only one redirection to https and /dataset
+      vars:
+        url: http://catalog-next.data.gov
+      block:
+        - name: look for expected redirection
+          uri:
+            url: "{{ url }}"
+            status_code: 302
+            follow_redirects: all
```

Comment:
This won't work. If you follow redirects, then you'll never see a status code of 302. You want `follow_redirect: none`.